### PR TITLE
Tests for source overflow on memcopy

### LINKED ISSUE #56: Test overflow from source on mem copy
There are currently no tests that attempt to overflow the source on memcopy by e.g. copying a massive amount of data. The following opcodes are likely vulnerable right now in Hera (see changes in https://github.com/ewasm/hera/pull/273):

- [x] return
- [x] revert
- [x] call*
- [x] create
- [x] log (#58)
- [ ] externalCodeCopy? (https://github.com/ewasm/hera/pull/273#issuecomment-408645978)

### PR DESCRIPTION
**Pull request by lrettig**
Closes #56 
Requires https://github.com/ewasm/hera/pull/273
Requires https://github.com/ethereum/cpp-ethereum/pull/5142 (which was merged but getting error about EVMC version when I try to run from the develop branch)

### CODE REVIEW

**Comment by axic** [comment]
Hera stuff merged.

**Comment by lrettig** [comment]
Thanks @axic. This one still needs some work (hence WIP tag), will finish up ASAP.

**Comment by lrettig** [comment]
This is ready for review, @hugo-dc could you take a look? Thanks

**Comment by hugo-dc** [comment]
@lrettig please take a look at `revertVeryLong`, `callOOB1`, `callOOB2`, `returnVeryLong`, seems those test cases need to be refilled.

**Comment by hugo-dc** [comment]
... also `callVeryLong`

File: src/GeneralStateTestsFiller/stEWASMTests/returnVeryLongFiller.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "create" (func $create (param i32 i32 i32 i32) (result i32)))
+          (memory 1)
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+          (func $main
+            (call $create
```
Comment (hugo-dc) [inline src/GeneralStateTestsFiller/stEWASMTests/returnVeryLongFiller.yml]: you are not using `return` in this test case.

Comment (lrettig) [reply src/GeneralStateTestsFiller/stEWASMTests/returnVeryLongFiller.yml]: Good catch! 🙏

**Comment by lrettig** [comment]
@hugo-dc updated, could you give this another shot? Thanks.

**Comment by lrettig** [comment]
Hugo says okay, merging :)

**Pull request merged**